### PR TITLE
creation of database models

### PR DESCRIPTION
**Pull request by avyavkumar**

### CODE REVIEW

File: mapping/models.py [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
+    department = models.CharField(choices=DEPARTMENT)
+    task = models.CharField(max_length=100)
+    subtask = models.CharField(max_length=100)    
+
+    def __str__(self):
+        return self.id
+
+class TaskDesc(models.Model):
+    STATUS = (
+        (1),(2),(3),(4),(5),
```
Comment (gnarula) [inline mapping/models.py]: Choices should be a tuple of tuples https://docs.djangoproject.com/en/1.8/ref/models/fields/

File: mapping/models.py [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+    )
+
+    task_dir = models.ForeignKey(TaskDir)
+    date = models.DateField()
```
Comment (gnarula) [inline mapping/models.py]: Change to a DateTimeField instead and set auto_now=True

File: mapping/models.py [hunk 3]
```diff
@@ -1,3 +1,38 @@
...
+    description = models.CharField(max_length=500)
+    status = models.IntegerField(choices=STATUS)
+
+    def __str__(self):
+        return self.taskID
```
Comment (gnarula) [inline mapping/models.py]: that's not valid. return self.task_dir.subtask instead

File: mapping/models.py [hunk 4]
```diff
@@ -1,3 +1,38 @@
...
+    # add other fields/metadata here 
+    
+    def __str__(self):
+        return self.name
+
+class TaskDir(models.Model):
+    DEPARTMENT = (
+        ('Health'),
+        ('Organisation Development', 'OD'),
+        ('Disaster Management', 'DM'),
```
Comment (gnarula) [inline mapping/models.py]: Reverse the sequence i.e ('DM', 'Disaster Management') and provide the second field for 'Health'

**Comment by avyavkumar** [comment]
renewed stuff

**Pull request merged**